server/utils/helpers.ts

```diff
@@ -62,3 +63,19 @@ export const formatAnomalyNumber = (num: number): string => {
     ? num.toFixed(2)
     : num.toExponential(2);
 };
+
+const PERMISSIONS_ERROR_PATTERN = /no permissions for \[(.+)\] and User \[name=(.+), backend_roles/;
+
+export const NO_PERMISSIONS_KEY_WORD = 'no permissions';
+
+export const prettifyErrorMessage = (rawErrorMessage: string) => {
+  if (isEmpty(rawErrorMessage)) {
+    return 'Unknow error is returned.';
+  }
+  const match = rawErrorMessage.match(PERMISSIONS_ERROR_PATTERN);
+  if (isEmpty(match)) {
+    return rawErrorMessage;
+  }
+  //@ts-ignore
+  return `User ${match[2]} has no permissions to [${match[1]}].`;
```

Comment:
minor: if we add the bottom message in an `else` block can we remove the `//@ts-ignore`, since `match` won't be empty?

Reply:
Sure. I can change to that.